tailwind.config.js

```diff
@@ -9,17 +9,17 @@ export default {
       current: 'currentColor',
       black: {
```

Comment:
Revieiwng the [communications guide](https://www.hsph.harvard.edu/communications-guide/design/colors/) again, it seems like there's a Shade 01/Shade 02, as well as a Tint 01/Tint 02 that might be better than what we're currently using here with the black gradient. I'm thinking we can also include the Harvard Chan Gray as part of the extended colors and implement that throughout the project.